src/views/AuthorityView/AuthorityView.js

```diff
@@ -181,10 +181,14 @@ const AuthorityView = ({
   });
 
   useEffect(() => {
+    if (isCentralTenantPermissionsLoading) {
+      return;
+    }
+
     if (!marcSource.isLoading && !authority.isLoading) {
       paneTitleRef.current?.focus();
     }
-  }, [marcSource.isLoading, authority.isLoading]);
+  }, [isCentralTenantPermissionsLoading, marcSource.isLoading, authority.isLoading]);
```

Comment:
Why add a new conditional? Just use the existing one that seems to check similar conditions:
```
if (!marcSource.isLoading && !authority.isLoading && !isCentralTenantPermissionsLoading) {
  paneTitleRef.current?.focus();
}
```